### ISSUE TITLE
NNPDEHan details: u0 and BatchNorm

### ISSUE THREAD

**Issue by luweizheng** [SciML/NeuralPDE.jl#525]
Hi,

In the original implementation(https://github.com/frankhan91/DeepBSDE/) of NNPDEHan, they use the neural network to model σᵀ∇u. This project use two neural networks, u0 and σᵀ∇u. Isn't σᵀ∇u the gradient of u0?
 
And in the original paper, they use BatchNorm after dense layer. But in this project (https://github.com/SciML/NeuralPDE.jl/blob/master/test/NNPDEHan_tests.jl) , authors do not use BatchNorm. 

Just curious why this project design the network different from the original papers? Any reasons?

**Comment by ChrisRackauckas** [comment]
>And in the original paper, they use BatchNorm after dense layer. But in this project (https://github.com/SciML/NeuralPDE.jl/blob/master/test/NNPDEHan_tests.jl) , authors do not use BatchNorm.

That has nothing to do with the design. Users can pass any neural network that they want. The implementation is agnostic to the neural network.

> In the original implementation(https://github.com/frankhan91/DeepBSDE/) of NNPDEHan, they use the neural network to model σᵀ∇u. This project use two neural networks, u0 and σᵀ∇u. Isn't σᵀ∇u the gradient of u0?

It's been awhile since I looked at this. But if you don't have a network for u0, wouldn't you not have a direct representation of the solution value for the control action?

**Comment by luweizheng** [comment]
Thanks for your reply.

Users can use any network architecture, they can choose to use BatchNorm or not.

And for NNPDEHan, we can use neural network to model u0 and σᵀ∇u. We do not need to stay same with the original implementation.
